### PR TITLE
Display System Serial and System Product in FreeNAS when using hardware from iX.

### PR DESCRIPTION
**Pull request by 650elx**
Ticket: 28369

### CODE REVIEW

File: gui/templates/system/system_info.html [hunk 1]
```diff
@@ -93,7 +93,7 @@ <h1>{% trans "System Information" %}</h1>
         <th>{% trans "Load Average" %}</th>
         <td class="descr">{{ local.loadavg }}</td>
     </tr>
-    {% if not is_freenas %}
+    {% if system_manufacturer == 'iXsystems' %}
```
Comment (william-gr) [inline gui/templates/system/system_info.html]: I think it should be ```local.system_manufacturer```?

Comment (650elx) [reply gui/templates/system/system_info.html]: Yes; And I have a correct version on my testing system. It's a typo.

Comment (william-gr) [reply gui/templates/system/system_info.html]: I would suggest mounting your git repo inside your testing system using NFS, that will prevent issues like that ; )

**Comment by 650elx** [comment]
rebuild

**Pull request merged**